src/data/organizations.js

```diff
@@ -0,0 +1,26 @@
+export default {
+  org1: {
+    ID: 'org1',
+    name: 'Organization 1',
+  },
+  org2: {
+    ID: 'org2',
+    name: 'Organization 2',
+  },
+  org3: {
+    ID: 'org3',
+    name: 'Organization 3',
+  },
+  org5: {
+    ID: 'org5',
+    name: 'Organization 5',
+  },
+  org6: {
+    ID: 'org6',
+    name: 'Organization 6',
+  },
+  org7: {
```

Comment:
Is having seven orgs necessary? I think just 2 should be sufficient for now, no?